sevntu-checks/.ci/eclipse-compiler-javac.sh

```diff
@@ -30,7 +29,10 @@ if [ ! -f $ECJ_PATH ]; then
     cd ..
 fi
 
-wget https://github.com/checkstyle/checkstyle/blob/checkstyle-$2/config/org.eclipse.jdt.core.prefs
+wget https://raw.githubusercontent.com/checkstyle/checkstyle/checkstyle-$2/config/org.eclipse.jdt.core.prefs
```

Comment:
github.com -> raw.githubusercontent.com

We need plain text file, not html. The check was not working before this change.